tools/pythonpkg/tests/arrow/test_nested_arrow.py

```diff
@@ -0,0 +1,80 @@
+import duckdb
+try:
+    import pyarrow as pa
+    import pyarrow.parquet
+    import numpy as np
+    can_run = True
+except:
+    can_run = False
+
+def compare_results(query):
+    true_answer = duckdb.query(query).fetchall()
+    t = duckdb.query(query).arrow()
+    from_arrow = duckdb.from_arrow_table(duckdb.query(query).arrow()).fetchall()
+
+    assert true_answer == from_arrow
+
+class TestArrowNested(object):
+    def test_lists_basic(self,duckdb_cursor):
+        if not can_run:
+            return
+        
+        #Test Constant List
+        query = duckdb.query("SELECT a from (select list_value(3,5,10) as a) as t").arrow()['a'].to_numpy() 
```

Comment:
Could you add some more tests:
* Complex nested structures, e.g.:
* `LIST[LIST]`
* `LIST[LIST[LIST[LIST[LIST[INTEGER]]]]]]`
*`STRUCT[LIST, LIST, STRUCT[INTEGER, LIST]]`
* Complex nested structures with different list types (i.e. same as above, but one list is a large list, another list is a fixed-size list, etc
* Tests for converting multiple lists to/from Arrow with NULL values and/or strings

Reply:
Line 74 does a list of lists, 
Cant really do structs since they are not implemented on this PR ( I have it in a branch and I already test the map type which is essentially a struct with lists :-)

I'll add the other ones.